es-models/gene_expression/gene_expression.mapping.yaml

```diff
@@ -0,0 +1,31 @@
+_source:
```

Comment:
It might be interesting to also enable `_size`, but I don't think it matters much either way. This index won't have a document size explosion.